falcon/request.py

```diff
@@ -167,6 +167,13 @@ class Request(object):
             all the values in the order seen.
 
         options (dict): Set of global options passed from the API handler.
+
+        cookies (dict):
+            A dict of name/value cookie pairs.
+
+    See Also:
```

Comment:
I think I'd rather just move this reference to L172, i.e.:

`A dict of name/value cookie pairs. See also: :ref:`Getting Cookies <getting-cookies>`